README.rst

```diff
@@ -114,3 +114,10 @@ the world. The following links should get you started:
 `<https://docs.saltstack.com/en/latest/topics/development/index.html>`_
 
 `<https://docs.saltstack.com/en/develop/topics/development/pull_requests.html>`_
+
+License
+=======
+
+SaltStack is licensed by Salt under the Apache license. Please see the LICENSE
```

Comment:
IMHO, this should explicitly say that it's the `2.0` version of the Apache license.

Reply:
Makes sense to me!